src/modules/universal-discovery/components/content-meta-preview/content.meta.preview.component.js

```diff
@@ -155,10 +200,17 @@ export default class ContentMetaPreviewComponent extends Component {
             <div className="c-meta-preview__wrapper">
                 <h1 className="c-meta-preview__title">{labels.title}</h1>
                 <div className="c-meta-preview" style={{ maxHeight: `${this.props.maxHeight - 64}px` }}>
-                    <div className="c-meta-preview__content-type">{this.renderIcon()} {contentTypeName}</div>
+                    <div className="c-meta-preview__type-and-bookmark-wrapper">
```

Comment:
I'm sorry but I don't like this class name. Can you create something better?

Reply:
I don't know what name choose for this class, `c-meta-preview__meta-top-wrapper`?